Civi/AIP/Processor/Api3.php

```diff
@@ -52,7 +52,11 @@ public function processRecord($record)
 
     // 4) Run the API call
     $this->log("Call API {$entity}.{$action} with parameters hash {$call_hash}", 'debug');
-    \civicrm_api3($entity, $action, $call_parameters);
+    $result = \civicrm_api3($entity, $action, $call_parameters);
+    if($this->getConfigValue('log/apicall') == '1') {
```

Comment:
I would prefer
```
if (!empty($this->getConfigValue('log/apicall'))) {
```
Or is the a different meaning planned for the different values of this config option?